features/adb-openshift.feature

```diff
@@ -0,0 +1,81 @@
+Feature: Command output from various OpenShift related commands in ADB
+  service-manager should return the correct output from commands affecting OpenShift in ADB
+
+  @ci-only
+  Scenario Outline: Boot and execute commands
+    Given box is <box>
+    And provider is <provider>
+    And a file named "Vagrantfile" with:
+    """
+    begin
+      require 'vagrant-libvirt'
+    rescue LoadError
+      # NOOP
+    end
+
+    Vagrant.configure('2') do |config|
+      config.vm.box = '<box>'
+      config.vm.box_url = 'file://../../.boxes/<box>-<provider>.box'
+      config.vm.network :private_network, ip: '<ip>'
+      config.vm.synced_folder '.', '/vagrant', disabled: true
+      config.servicemanager.services = 'docker, openshift'
+
+      config.servicemanager.openshift_image_tag = 'v1.2.1'
```

Comment:
So these three lines are the only remaining mystery to me. Why are they needed and won't they be problematic when a new version of adb and/or openshift image become available. Where or what is the issue here - @budhrg, @praveenkumar